src/piece.cpp

```diff
@@ -110,6 +110,14 @@ namespace {
       p->stepsCapture = {2 * SOUTH_WEST, 2 * SOUTH_EAST, 2 * NORTH_WEST, 2 * NORTH_EAST};
       return p;
   }
+  PieceInfo* dabbaba_piece() {
+      PieceInfo* p = new PieceInfo();
+      p->name = "dabbaba";
+      p->betza = "D";
+      p->stepsQuiet = {2 * SOUTH, 2 * WEST, 2 * EAST, 2 * NORTH};
+      p->stepsCapture = {2 * SOUTH, 2 * WEST, 2 * EAST, 2 * NORTH};
```

Comment:
I think:

- "quiet" = not capturing
- "capture" = capturing

- "steps" = jumps like a knight or alfil
- "slider" = sliding like a rook or bishop
- "hopper" = jump-and-capture like a checkers piece, but  I'm not sure what `hopperQuiet` would mean

Reply:
Yes, your understanding is correct. Hopper generally refers to pieces that need a screen/hurdle to jump, but does not necessarily imply a capture move, although the cannon might be the most well known example. So `hopperQuiet` refers to a cannon-like move where the target square is empty, e.g., like for the cannon in Janggi.